packages/honeycomb-opentelemetry-web/test/global-errors-instrumentation.test.ts

```diff
@@ -37,6 +37,10 @@ describe('Global Errors Instrumentation Tests', () => {
         'exception.type': 'Error',
         'exception.message': 'Something happened',
         'exception.stacktrace': expect.any(String),
+        'exception.structured_stacktrace.columns': expect.any(Array),
```

Comment:
Ah I see we have these unit tests but it would be great to see a unit test that tests the specific data transform logic too if possible!